File: ComInLanDroid/cominlan/src/main/java/com/onballgroup/cominlan/model/protocol/IServerProtocol.java
package com.onballgroup.cominlan.model.protocol;

import com.onballgroup.cominlan.model.Base.IJson;

/**
 * Created by Phong Le on 4/20/2016.
 */
public interface IServerProtocol extends IJson {
    ServerCommand getCommand();
}

File: ComInLanWindows/ComInLan/Model/Protocol/IServerProtocol.cs
using ComInLan.Model.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ComInLan.Model.Protocol
{
	public interface IServerProtocol : IJson
	{
		ServerMessage Message { get; }
	}
}
